composer.json

```diff
@@ -9,7 +9,10 @@
     }
   ],
   "require": {
-    "php": ">=7.1"
+    "php": ">=7.1",
+    "spryker/product-page-search": "^3.0.0",
```

Comment:
@SpiderDan98 did you consider all possible versions?

Reply:
@daniel-rose Yes now